docker/importer/importer.py

```diff
@@ -55,6 +56,17 @@ def _is_vulnerability_file(source_repo, file_path):
   return file_path.endswith(source_repo.extension)
 
 
+def aestnow() -> datetime:
+  """Get the current AEST time"""
+  # - First make datetime timezone aware
+  # - Then set timezone to AEST without DST (+10)
+  # - Then make datetime timezone unaware again to keep
+  #   compatibility with ndb
+  return utcnow().replace(tzinfo=tz.UTC).astimezone(
+      tz.tzoffset('AEST_NO_DST',
```

Comment:
What does dateutil.tz give us if we still need to specify the offsets ourself here?

Can we just use https://docs.python.org/3/library/datetime.html#timezone-objects instead?

Reply:
Good point, removed dateutil now.